### ISSUE TITLE
Weblate email last wednesday

### ISSUE THREAD

**Issue by trebmuh** [captnfab/PianoBooster#289]
Hi @louis-barman .

I'm checking my email box after 2 weeks off-the-internet, and I see a message from weblate about: "Does not meet the conditions for free hosting". Did you received those as well?

**Comment by pianobooster** [comment]
Yes I received it. but I don't know what to do about it. I had quite a good
look but could not spot which error was causing it. I emailed Allan Nordhøy
<epost@anotheragency.no> who asked to be an admin but he has not responded.
would it help If I made you an admin. Can you see what is wrong?

Hello Allan,


Do you please know what is causing this alert (See the emails below)? They
say "Does not meet libre hosting conditions." Does that mean they remove
PianoBooster from weblate?


Thanks

Louis

**Comment by trebmuh** [comment]
> Yes I received it. but I don't know what to do about it. I had quite a good look but could not spot which error was causing it. I emailed Allan Nordhøy `<epost@anotheragency.no>` who asked to be an admin but he has not responded. would it help If I made you an admin.

Not sure. Maybe try to contact weblate people directly?

> Can you see what is wrong?

No, I don't know. I've received email such as this one for a few (4 or 5) projects I'm doing translations on Weblate for. So that's not only pianobooster.

Olivier

**Comment by comradekingu** [comment]
Must have missed that e-mail. On it now though.

According to
https://hosted.weblate.org/guide/pianobooster/pianobooster/
and
https://hosted.weblate.org/guide/pianobooster/boostermusic/

https://docs.weblate.org/no/latest/admin/continuous.html?highlight=webhooks#automatically-receiving-changes-from-github should be set up.

Outside of that, I turned on shared translation memory in https://hosted.weblate.org/settings/pianobooster/#workflow

Edit: Also used `https://github.com/captnfab/PianoBooster.git` as Push-URL in https://hosted.weblate.org/settings/pianobooster/boostermusic/#vcs
Language consistency plugin turned on. rudimentary translator instructions added.

**Comment by lfzawacki** [comment]
Hello, I wanted to start working on a PT-BR translation but it seems there's a problem with the translation repository and weblate. The following error:

![Selection_355](https://user-images.githubusercontent.com/33211/127943823-bbb0cf55-5b12-413d-b08e-a4710eecd0c2.png)

I presume this has to do with this issue, is there some update needed on your side to make this work?

**Comment by comradekingu** [comment]
I think it needs to change source code archive from
https://github.com/captnfab/PianoBooster.git to  https://github.com/pianobooster/PianoBooster 

And push address from
https://github.com/captnfab/PianoBooster.git to https://github.com/pianobooster/PianoBooster.git

Removed translations/music_blank.ts as template for new translations as it doesn't exist

And https://github.com/pianobooster/PianoBooster/pull/302 merged, because there are two duplicated Bosnian language codes.

From the error it wouldn't hurt to add https://github.com/weblate/
Can't do that myself, since I don't have access to this repo.

**Comment by louis-barman** [comment]
Ok Great thanks for that. I could not see what was wrong. I needed to put in a support request to try and get it resolved. But I'll try that suggestion first when I get back. I am away at the moment.
